rel-1.0.0-preview2.1/debian/core-deps/hooks/post_push

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+# Issue - https://github.com/docker/hub-feedback/issues/384#issuecomment-214508878
+# Docker Hub emits an invalid .dockercfg.  Remove the outer auths element.
+if [[ $DOCKERCFG == {\"auths\":* ]]; then
+    jsonStartIndex=$(expr index "$DOCKERCFG" :)
+    length=$((${#DOCKERCFG} - $jsonStartIndex - 1))
+    echo ${DOCKERCFG:jsonStartIndex:length} > /root/.dockercfg
+fi
+
+
+echo "Pushing core images"
```

Comment:
This should be "image" instead of "images". Same for the onbuild image in rel-1.0.0-preview2.1/debian/hooks/post_build.